app/src/main/java/comp3350/mealbuddy/objects/Account.java

```diff
@@ -0,0 +1,22 @@
+package comp3350.mealbuddy.objects;
+
+import java.util.ArrayList;
+
+public class Account {
+
+    public UserInfo user;
+    private ArrayList<Goal> goals;
+    public Account(UserInfo user) {
```

Comment:
I'm not sure but do we need a way to determine which food was eaten by this account as opposed by other accounts?  Or is this all handled in data base layer with having a field like "user" as a one of the keys?  If so we need to pass user name when getting food from db stub.  But then we might be introducing logic that we dont want into the db layer so that might not be a good idea...how does the sample get a courses just from a student in the sample?

Reply:
Interesting question. I think this can be tweaked when we implement it, hard to plan for it.

If you think it is important, we can talk about this in slack and i'll fix it appropriately

Reply:
Yeah I'm not sure how else we would link the food eaten with the account.  Even just thinking immediate short term we need a way to see what food is eaten that day by the user.  We can decide next iteration how we want to do it once we introduce a data base.

Reply:
This has been resolved with the addition of Days.